index.html

```diff
@@ -1,277 +1,268 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
+
+  <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
     <title>Alex Portfolio</title>
-</head>
-<body>
+  </head>
+
+  <body>
     <!--Principal nav-->
     <header>
```

Comment:
- [ ] your menu and should be `display: none;` by default  in the desktop version but as I can see it appearing in the desktop version so please try to fix that